testApi/src/main/scala/africa/absa/testing/scapi/ScAPIRunner.scala

```diff
@@ -47,7 +47,11 @@ object ScAPIRunner {
       case Failure(exception) => throw exception
     }
 
-    Logger.setLevel(if (cmd.debug) Level.DEBUG else Level.INFO)
+    cmd match {
+      case c if c.trace => Logger.setLevel(Level.TRACE)
+      case c if c.debug => Logger.setLevel(Level.DEBUG)
+      case _ => Logger.setLevel(Level.INFO)
+    }
```

Comment:
The log level is normally set by `log4j.properties` like

```properties
# Set root logger level to TRACE and its only appender to A1.
log4j.rootLogger=TRACE, A1

# A1 is set to be a ConsoleAppender.
log4j.appender.A1=org.apache.log4j.ConsoleAppender

# A1 uses PatternLayout.
log4j.appender.A1.layout=org.apache.log4j.PatternLayout
log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n

# Set the logger level for org.apache.logging.log4j to TRACE
log4j.logger.org.apache.logging.log4j=TRACE
```


Why have this? Is it controlling more then just the Log4j log level?

Reply:
There were no plan to provide logging level control to user.

- default level is INFO
- user can allow DEBUG or TRACE levels, where TRACE level is very hard to read but can be useful when DEBUG does not help

Reply:
Blocking prepared in commit 8f4ca71bef4d8b8397e35a9f13f2cb2077a8c28b.